File: TOMIMSV4/Entity/Mantenimientos/Producto/Producto_Color/clsBeColor.vb
Public Class clsBeColor
    Implements ICloneable

    Public Property IdColor() As Integer = 0
    Public Property Nombre() As String = ""
    Public Property CodigoHex() As String = ""
    Public Property IdPropietario() As Integer = 0
    Public Property Fec_agr() As Date = Now
    Public Property User_agr() As String = ""
    Public Property Fec_mod() As Date = Now
    Public Property User_mod() As String = ""

    Public Property Activo() As Boolean = False
    Public Property IsNew() As Boolean = False
    Public Property Codigo() As String = ""
    Public Property Propietario As New clsBePropietarios


    Sub New()
        '#EJC: Add your constructor here... 
    End Sub

    Public Function Clone() As Object Implements System.ICloneable.Clone
        Return MyBase.MemberwiseClone()
    End Function

End Class

File: WMS.EntityCore/Datos_Maestros/clsBeColor.cs
using System;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations.Schema;

namespace WMSWebAPI.Be
{
    public class clsBeColor : ICloneable
    {
        [Column("IdColor")]
        [DisplayName("IdColor")]
        public int IdColor { get; set; } = 0;

        [Column("Nombre")]
        [DisplayName("Nombre")]
        public string Nombre { get; set; } = "";

        [Column("CodigoHex")]
        [DisplayName("CodigoHex")]
        public string CodigoHex { get; set; } = "";

        [Column("IdPropietario")]
        [DisplayName("IdPropietario")]
        public int IdPropietario { get; set; } = 0;

        [Column("fec_agr")]
        [DisplayName("fec_agr")]
        public DateTime Fec_agr { get; set; } = DateTime.Now;

        [Column("user_agr")]
        [DisplayName("user_agr")]
        public string User_agr { get; set; } = "";

        [Column("fec_mod")]
        [DisplayName("fec_mod")]
        public DateTime Fec_mod { get; set; } = DateTime.Now;

        [Column("user_mod")]
        [DisplayName("user_mod")]
        public string User_mod { get; set; } = "";

        [Column("activo")]
        [DisplayName("activo")]
        public bool Activo { get; set; } = false;

        public clsBeColor() { }

        public object Clone()
        {
            return this.MemberwiseClone();
        }
    }
}
